launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml

```diff
@@ -35,13 +37,14 @@
     <arg name="image_number" value="$(var image_number)"/>
   </include> -->
 
-  <!-- Pointcloud map filter -->
+  <!-- detection preprocess module -->
   <group>
-    <include file="$(find-pkg-share compare_map_segmentation)/launch/voxel_based_compare_map_filter.launch.xml" if="$(var use_pointcloud_map)">
-      <arg name="input" value="$(var input/obstacle_segmentation/pointcloud)"/>
-      <arg name="input_map" value="/map/pointcloud_map"/>
-      <arg name="output" value="pointcloud_map_filtered/pointcloud"/>
-      <arg name="distance_threshold" value="0.5"/>
+    <include file="$(find-pkg-share perception_launch)/launch/object_recognition/detection/detection_preprocess.launch.py" if="$(var use_pointcloud_map)">
+      <arg name="input_topic" value="$(var input/obstacle_segmentation/pointcloud)"/>
+      <arg name="use_intra_process" value="true"/>
+      <arg name="use_multithread" value="true"/>
+      <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
+      <arg name="container_name" value="$(var pointcloud_container_name)"/>
```

Comment:
can you add about the output arg?
```
 <arg name="output" value="pointcloud_map_filtered/pointcloud"/>
```

Reply:
Fixed at [f0e6e5e](https://github.com/autowarefoundation/autoware.universe/pull/961/commits/f0e6e5e0f396b4c81b6c6d2e6ca85550944d4fb3)